i18n/locales/urdu/translations.json

```diff
@@ -1,76 +1,80 @@
 {
   "buttons": {
-    "forum": "Forum",
-    "donate": "Donate",
-    "load-more-articles": "Load More Articles"
+    "forum": "فورم",
+    "donate": "عطیہ کریں",
+    "load-more-articles": "مزید آرٹیکلز لوڈ کریں"
   },
   "search": {
     "label": "Search",
-    "accessible-name": "Submit your search query",
-    "placeholder": "Search 8,000+ tutorials",
-    "see-results": "See all results for {{ searchQuery }}",
-    "no-tutorials": "No tutorials found"
+    "accessible-name": "اپنی search query لکھیں ",
+    "placeholder": " 8,000+ tutorials",
```

Comment:
The number 8,000+ refers to the number of English tutorials / articles that have been published. Until a substantial number of articles have been published in Urdu, it's probably better to use a more generic term like "Search tutorials".

Reply:
> @scissorsneedfoodtoo do you know why these tests are failing?

According to logs, it seems like a connectivity timeout issue with localhost. But I can't see how it correlates with the changes in translations.

Logs:
`http://localhost:8080 timed out on retry 1228 of 41, elapsed 1230157ms, limit 1230000ms
Error: connect ECONNREFUSED [127](https://github.com/freeCodeCamp/news/runs/6117130638?check_suite_focus=true#step:6:127).0.0.1:8080`